opensoundscape/datasets.py

```diff
@@ -330,3 +330,57 @@ def __getitem__(self, item_idx):
             labels = np.array([row[self.label_column]])
             return {"X": X, "y": torch.from_numpy(labels)}
         return {"X": X}
+
+# class PredictionDataset(torch.utils.data.Dataset):
```

Comment:
Do you think we should keep this as a separate class, or encourage users to use an all-purpose dataset like `SingleTargetAudioDataset` (#179)? I could see pros and cons of either approach